packages/react/src/components/TextInput/PasswordInput-test.js

```diff
@@ -1,213 +0,0 @@
-import React from 'react';
-import PasswordInput from './PasswordInput';
-import { mount, shallow } from 'enzyme';
-
-const prefix = 'cds';
-
-describe('PasswordInput', () => {
-  describe('renders as expected', () => {
-    const wrapper = mount(
-      <PasswordInput
-        id="test"
-        className="extra-class"
-        labelText="testlabel"
-        helperText="testHelper"
-        showPasswordLabel="Show password"
-        hidePasswordLabel="Hide password"
-      />
-    );
-
-    const passwordInput = () => wrapper.find('input');
-
-    describe('input', () => {
-      it('renders as expected', () => {
-        expect(passwordInput().length).toBe(1);
-      });
-
-      it('should accept refs', () => {
```

Comment:
Is there a test for refs in the new file? I think we'd just want to assert that it's placed on the input element. Something like this might work:

```js
  it('should support a `ref` that is placed on the <input> element', () => {
    const ref = jest.fn();
    render(
        <PasswordInput
          labelText="PasswordInput label"
          ref={ref}
        />
      );
    expect(ref).toHaveBeenCalledWith(screen.getByLabelText('PasswordInput label'));
  });
```

I found quite a few spots in the codebase we already do similar versions of this - do a global search for `` `ref` `` in `*-test.js` files for more examples.